runtime_config_linux.go

```diff
@@ -195,7 +195,7 @@ type Network struct {
 	// Set class identifier for container's network packets
 	ClassID *uint32 `json:"classID"`
 	// Set priority of network traffic for container
-	Priorities []InterfacePriority `json:"priorities"`
+	Priorities []*InterfacePriority `json:"priorities,omitempty"`
```

Comment:
For this PR just adding `omitempty` here should do. Kindly revert the pointer changes.